iset.mm

```diff
@@ -128386,6 +128386,16 @@ The circle constant (tau = 2 pi)
     EWSNWBWTWEXANZABVAOLEDLMDXJVELQVBXKWBWGWEWINWBWHWEWJNWBWERTVDWBXFVPXHVQWBXE
     ACFWBABXCXDVCSWBXGBCFWBBXDVFSUSUTVGWAVMVSEDVTWFJABVHCVSVIVJVKVL $.
 
+  $( A number greater than 1 divides an integer minus 1 iff the integer modulo
```

Comment:
```suggestion
  $( An integer greater than 1 divides an integer minus 1 iff the integer modulo
```

Reply:
First of all, if we are going to revise this wording we should do it in set.mm too.

Secondly, I think if we are going to change this it would involve changing more of the text. The comment currently refers to ` N ` as "a number" and ` A ` as "an integer" and uses each of those terms twice.

Maybe "An integer greater than one divides another integer minus one iff the second integer modulo the first integer is one." which I guess is kind of like how some other nearby theorems seem to describe it.

Don't know if @avekens has any ideas on this topic, especially since this theorem seems to be from only a bit more than a year old.

Reply:
> Maybe "An integer greater than one divides another integer minus one iff the second integer modulo the first integer is one." which I guess is kind of like how some other nearby theorems seem to describe it.

Sounds good.

Reply:
Thanks. But as usual, force pushing makes it harder for reviewers to check that you addressed their reviews, compared to a specific commit. Most contributors do the latter, see e.g., `git log --grep='review'`.  Then, the squash-and-merge allows not cluttering the history.

Reply:
> Don't know if @avekens has any ideas on this topic, especially since this theorem seems to be from only a bit more than a year old.

I can't remember if I have chosen the term "number" with any intention. Maybe because Norm often used "number" as short version of "real number" (see, for example, ~modfrac "The fractional part of a number ..."), but here we do not have real numbers. Maybe I was inspired by the letter `N`. Or maybe, it was used simply to distinguish the two involved numbers.
Nevertheless, the proposed comment is fine for me. An alternate version could have been:

> "A positive integer greater than one divides an integer minus one iff the integer modulo the positive integer is one."

Reply:
PS: Or I just copied the comment from ~modprm1div and removed "prime"...

Reply:
> Thanks. But as usual, force pushing makes it harder for reviewers to check that you addressed their reviews, compared to a specific commit. Most contributors do the latter, see e.g., `git log --grep='review'`. Then, the squash-and-merge allows not cluttering the history.

One of the tradeoffs is that if the final commit message is only written (well or edited anyway) at merge time, no one gets to review it.

Having said that, it doesn't matter very much to me. I've known projects which very much prefer the rebasing and meaningful (meaning reflecting the logical division rather than the history) commits at review time, but if people prefer the squashing at merge time, I'm OK with that too.

Reply:
As for wordings with "number" I like the way that reads in a lot of cases. The comment doesn't always need to state what kind of number it is. In this case it didn't work out as well but either suggested rewrite addresses the issue, so we should be good now.